core/drivers/crypto/caam/include/caam_hash.h

```diff
@@ -1,21 +1,24 @@
 /* SPDX-License-Identifier: BSD-2-Clause */
 /*
- * Copyright 2018-2019 NXP
+ * Copyright 2018-2021 NXP
  *
  * Brief   CAAM Hash manager header.
  */
 #ifndef __CAAM_HASH_H__
 #define __CAAM_HASH_H__
 
+#include <caam_jr.h>
+
 #ifdef CFG_NXP_CAAM_HASH_DRV
 /*
  * Initialize the Hash module
  *
  * @ctrl_addr   Controller base address
```

Comment:
You forgot to update the comment (same thing below)

Reply:
Fixed!